tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "allowJs": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true,
```

Comment:
Why would we want to skip type checking .d.ts files?

Reply:
Mostly to speed up things since we assume provided libraries' typedef is accurate. It's also suggested by the vue3 project creator.